index.html

```diff
@@ -0,0 +1,204 @@
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- Start Headline -->
+    <section class="headline container mt-3">
+      <h1 class="d-block mb-2">
+        "Hello Future Programmers"
+      </h1>
+      <img src="./imgs/headline.png" alt="Creatve Programmers and UI UX Designers" class="col-lg-9 col-sm-10 col-12"/>
+      <p class="ms-md-5 ms-3">
+        if you want to be a professional web developer. 
+        So it's time to take our course and become a professional developer. 
+        We are a group of professional teachers.
+      </p>
+    </section>
+    <!-- Start Main Program -->
+    <section class="main-program container-fluid text-light text-center mb-4 py-5">
+      <h2 class="mb-5">
+        Main Program
+      </h2>
+      <div class="row">
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/design.png" class="card-img-top" alt="design">
+            <div class="card-body">
+              <h5 class="card-title">UI & UX design</h5>
+              <p class="card-text">
+                We will teach you how to design and become a professional designer.
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/html-css-js.png" class="card-img-top" alt="html-css-js">
+            <div class="card-body">
+              <h5 class="card-title">Web Development Basics</h5>
+              <p class="card-text">
+                you will learn web development basics which are HTML, CSS, and JS
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/react-redux.jpg" class="card-img-top" alt="react-redux">
+            <div class="card-body">
+              <h5 class="card-title">ReactJS & Redux</h5>
+              <p class="card-text">
+                We will teach you one of popular frameworks ReactJS and Redux to handle states
+                to make your work easer and professional
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-3 col-md-6 col-12 mt-2">
+          <div class="card flex-sm-row flex-row flex-lg-column ">
+            <img src="./imgs/node-mongo.png" class="card-img-top" alt="Ruby & Ruby on rails">
+            <div class="card-body pt-0">
+              <h5 class="card-title">Ruby On Rails</h5>
+              <p class="card-text">
+                Ruby on Rails makes it much easier and more fun. 
+                It includes everything you need to build fantastic applications,
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Work on the layout of this section, right now looks messy, and it's because your images are from different sizes and move the layout, set all the images at the same level as the original design, also the background color is incorrect. Please, fix it.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132544689-05fd1751-6d83-4e41-9ae8-a3a335cca836.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132544753-0c471b40-d361-445f-8133-66606d69e18a.png)

Reply:
Addressed this as well. Good job!